src/style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  background-color: #fff;
+  margin: 0;
```

Comment:
Suggestion: styles like `color`, `font-family` can be inherited. You can apply it directly to the `body`.
We need `margin: 0` and `box-sizing: border-box` with `*` to apply it for all elements at one time and to not repeat it every time.
`background-color` can also be applied to the `body`
So, the golden version will be: 
```css
* {
  margin: 0;
  box-sizing: border-box;
}
body {
  font-family: "Roboto", sans-serif;
  color: #000;
  background-color: #fff;
}
```